### PR TITLE
Docs: document labels for trusted clusters

### LINKED ISSUE #5583: Document cluster labels in the admin guide
## Details

https://goteleport.com/teleport/docs/admin-guide/ has no mention on cluster labels added in 5.0
The only "public docs" are https://github.com/gravitational/teleport/blob/master/CHANGELOG.md#cluster-labels

### Category
- [x] New Feature
- [ ] Improve Existing
- [ ] Remove Outdated

### Which platform(s)
- [ ] Teleport Open Source
- [ ] Teleport Cloud
- [x] Teleport Enterprise
- [ ] Other

### Which component(s)
- [ ] Server Access
- [ ] Kubernetes Access
- [ ] Application Access
- [ ] Database Access
- [ ] CLI Clients
- [ ] Web Clients
- [x] Other

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #5583

- Adds the `--labels` flag to `tctl tokens add` reference.
- Adds `Details` block on labels to Trusted Clusters doc.

### CODE REVIEW

File: docs/pages/management/admin/trustedclusters.mdx [hunk 1]
```diff
@@ -286,6 +286,12 @@ Use this token when defining a trusted cluster resource on a remote cluster.
 This command generates a Trusted Cluster join token. The token can be used
 multiple times and has an expiration time of 5 minutes.
 
+<Details title="Optional: Add labels">
+The `--labels` flag will add labels to the trusted cluster. This lets teams use
```
Comment (ptgott) [inline docs/pages/management/admin/trustedclusters.mdx:290]:
```suggestion
The `--labels` flag will add labels to the Trusted Cluster. This lets teams use
```

File: docs/pages/reference/cli.mdx [hunk 2]
```diff
@@ -1810,10 +1810,11 @@ $ tctl tokens add --type=TYPE [<flags>]
 
 | Name | Default Value(s) | Allowed Value(s) | Description |
 | - | - | - | - |
+| <nobr>`--format`</nobr> | none | `text`, `json`, `yaml` | Output format |
+| `--labels` | none | `text` | Sets token labels |
+| `--ttl` | 1h | relative duration like 5s, 2m, or 3h | Set expiration time for token |
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:1815]:
```suggestion
| `--ttl` | 1h | Duration like 5s, 2m, or 3h | Set expiration time for token |
```

**Review by ptgott — APPROVED** [review]
Approved with minor suggestions

File: docs/pages/reference/cli.mdx (hunk 2 above)
Comment (zmb3) [reply docs/pages/reference/cli.mdx:1815]: I wouldn't say "set expiration time" - you're not setting an expiration time (that would be like saying "expire at 4pm tomorrow") - you're setting a validity period.

Comment (alexfornuto) [reply docs/pages/reference/cli.mdx:1815]: FYI I didn't change it to say that, it's been the description for this flag as far back as the docs history goes. See L1816 where it was moved from. In any case, how about?

```suggestion
| `--ttl` | 1h | Duration like 5s, 2m, or 3h | Sets how long the token is valid for. |
```

**Review by zmb3 — APPROVED** [review]

**Pull request merged**